Code/GraphMol/SmilesParse/SmilesParse.h

```diff
@@ -127,12 +127,9 @@ inline std::unique_ptr<RDKit::RWMol> operator"" _smiles(const char *text,
 inline std::unique_ptr<RDKit::RWMol> operator"" _smarts(const char *text,
                                                         size_t len) {
   std::string smi(text, len);
-  RWMol *ptr = nullptr;
-  try {
-    ptr = SmartsToMol(smi);
-  } catch (const RDKit::MolSanitizeException &) {
-    ptr = nullptr;
-  }
+  // no need for exception handling here: SmartsToMol() doesn't do
```

Comment:
This comment might only make sense in the context of removing the exception handling :)

Reply:
It's there to make it clear that not having exception handling here (unlike in the `_smiles` version) wasn't an oversight.